### DISCUSSION TITLE
Help for integrations

### DISCUSSION THREAD

**Discussion by ganeshkumars10d** [braintree/braintree_android#608] [Q&A] [answered]
How to config braintree drop in ui with react native ,i have read the document for android , but i have no idea of where to config
 that java / kotlin code into my react native project can anyone guide me to solve this

**Comment by sshropshire** [comment, marked as answer]
Hi @ganeshkumars10d thanks for using the Braintree SDK for Android. We do not offer official support for react native at this time, however there are some [community libraries](https://www.npmjs.com/package/react-native-braintree-dropin-ui) that exist. I suggest reading the source of the projects to get a quick view of a react native integration.
